backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidInput.java

```diff
@@ -571,11 +575,7 @@ public boolean onKey (View v, int keyCode, android.view.KeyEvent e) {
 			app.getGraphics().requestRendering();
 		}
 
-		// circle button on Xperia Play shouldn't need catchBack == true
-		if (keyCode == Keys.BUTTON_CIRCLE) return true;

```

Comment:
You could now in theory do this:
````java
setCatchKey(Keys.BUTTON_CIRCLE, false);
````
which would break this workaround.
See https://badlogicgames.com/forum/viewtopic.php?f=11&t=1317
and https://github.com/dwaite/imame4all/issues/203

Therefore I think this line should stay unchanged.

Reply:
That's by intention. If someone needs to, the default behaviour can be changed by the developer.